cmake/Modules/FindComputeCpp.cmake

```diff
@@ -212,6 +218,27 @@ define_property(
   non-standards-conformant SYCL code to compile with ComputeCpp."
 )
 
+# Enable the usage of the ComputeCpp compiler driver. This removes the need
+# to explicitly generate the sycl integration header in a separate compiler pass.
+if(COMPUTECPP_USE_COMPILER_DRIVER)
+  # Check if the current system supports the compiler driver
+  set(isCompilerDriverSupported NO)
+
+  # For now, the compiler driver is only supported if the host compiler is either clang or gcc
```

Comment:
Is this to ensure that any additional compiler flags are recognised? You are going to override the `CXX` compiler, so I'm not sure I understand this requirement?